README.md

```diff
@@ -8,11 +8,11 @@
 
 A small Java library for talking to GitHub/GitHub Enterprise and interacting with projects.
 
-It supports authentication via simple access tokens, JWT endpoints and GitHub Apps (via private key).
+It supports authentication via simple access tokens, JWT endpoints, and GitHub Apps (via private key).
 
 It is also very light on GitHub, doing as few requests as necessary.
 
-This library is maintained by @spotify/gjc-maintainers. If you have any questions, issues or need a
+This library is maintained by @spotify/gjc-maintainers. If you have any questions, or issues or need a
```

Comment:
This additional `or` can be debated because `questions` and `issues` are for the verb `have` whereas the third item `need a review` has its own verb. If we actually decided to add this `or`, the used commas are confusing for me. I would also suggest to not introduce this change.